src/pal/prebuilt/idl/clrdata_i.cpp

```diff
@@ -14,15 +14,13 @@
 
 #pragma warning( disable: 4049 )  /* more than 64k source lines */
 
+#include <rpc.h>
```

Comment:
I didn't know you could do that! Thanks for the suggestion. I'll give that a try and rebase without the "moving header" commit. There were two files that I changed back to .c and I think I do want to know if that is okay.